app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -1,4 +1,4 @@
-- return unless %w(ems_cloud ems_infra ems_container ems_network ems_physical_infra).include?(controller_name)
+- return unless %w(ems_cloud ems_infra ems_container ems_network ems_physical_infra ems_block_storage ems_storage).include?(controller_name)
```

Comment:
Unnecessary change, we no longer use the angular forms for providers...

Reply:
removed